include/picongpu/plugins/Checkpoint.hpp

```diff
@@ -77,7 +77,7 @@ namespace picongpu
         {
             namespace po = boost::program_options;
             if(ioBackendsHelp.empty())
-                desc.add_options()("checkpoint", "plugin disabled [compiled without dependency HDF5 or Adios]");
+                desc.add_options()("checkpoint", "plugin disabled [compiled without dependency HDF5 or ADIOS2]");
```

Comment:
```suggestion
                desc.add_options()("checkpoint", "plugin disabled [compiled without dependency openPMD-api]");
```
I think that this message still comes from a time when we wrote to ADIOS1 and HDF5 directly. 
Do we still have ADIOS1? then `without dependency ADIOS1 or openPMD-api`

Reply:
No, ADIOS1 is effectively already removed. And this PR removes leftover mentions of it (that should have been removed before as well, but missed). So i think (the changed version of) this message is accurate now.